Assets/HoloToolkit-Examples/SpatialUnderstanding/SpatialUnderstanding-FeatureOverview/Scripts/AppState.cs

```diff
@@ -7,196 +7,198 @@
 using HoloToolkit.Unity.InputModule;
 using HoloToolkit.Unity.SpatialMapping;
 
-public class AppState : Singleton<AppState>, ISourceStateHandler, IInputClickHandler
+namespace HoloToolkit.Examples.SpatialUnderstandingFeatureOverview
```

Comment:
Seems like a lot of these changes are address in https://github.com/Microsoft/HoloToolkit-Unity/pull/432